packages/fast-components-react-msft/src/select/examples.data.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import { ComponentFactoryExample } from "@microsoft/fast-development-site-react";
+import schema from "./select.schema.json";
+import { Select, SelectProps } from "./index";
+import { SelectOptionProps } from "../select-option";
+import Documentation from "./.tmp/documentation";
+
+function selectOptionPropFactory(id: string): SelectOptionProps {
+    return {
+        id,
+        value: id,
+        role: "option",
+        displayString: "Option-" + id,
+    };
+}
+
+const examples: ComponentFactoryExample<SelectProps> = {
+    name: "Select",
+    component: Select,
+    schema: schema as any,
+    documentation: <Documentation />,
+    detailData: {
+        placeholder: "Select an option",
+        children: [
+            {
+                id: "@microsoft/fast-components-react-base/select-option",
```

Comment:
this will need to be updated after you make the schema change from base to msft

Reply:
fixed